platform/core/src/services/DisplaySetService/DisplaySetMessage.ts

```diff
@@ -0,0 +1,87 @@
+const CODES = {
```

Comment:
We should check with @sedghi if this is a good location for this file.

Reply:
CODES can be static property of the DisplaySetMessageList

Reply:
like we have for events

Reply:
Done